docs/readmes/lte/dev_notes.md

```diff
@@ -48,6 +48,72 @@ file is generated by mapping the configuration values from `gateway.mconfig` and
 `/etc/systemd/system` directory of the VM at the time of provisioning. You need
 to manually sync changes to these files between guest and host OS.
 
+- Systemd configuration files contain interdependencies of Magma services regarding the start, stop, and restart of services. Therefore, a systemd configuration contains several options to specify these dependencies.
+- The options `Before` and `After` define the order in which services are started and stopped. If a service gets started, the services listed in `Before` and `After` will also be started. If one of the other services gets stopped or its start fails, the service will be stopped. The following dependency graph depicts this order for Magma services.
+
+    ```mermaid
+    graph TD
+        SubscriberDB --> MobilityD
+
+        OVS --> MobilityD
+        MobilityD --> MME
+
+        MobilityD --> PipelineD
+        EnvoyController --> PipelineD
+        EnvoyController --> Envoy
+
+        OVS --> PipelineD
+        PipelineD --> MME
+
+        PipelineD --> ConnectionD
+        PipelineD --> LiAgent
+
+        OVS --> ConnectionD
+        ConnectionD --> MME
+
+        SctpD --> MME
+        SessionD --> MME
+    ```
+
+- The options [Wants](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Wants=) and [Requires](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Requires=) specify a set of services that will be started if the configuring service starts. If the listed services fail to start, this has no impact on the service start in the case of `Wants`. Considering `Requires`, the service will not be started if the option `After` is set on the failing service.
+The subsequent figure illustrates both configuration options for Magma services. For example, `EnvoyController → Envoy` is read as "If EnvoyController starts, require to start Envoy as well." and `MobilityD → SubscriberDB` as "If MobilityD starts, a start of SubscriberDB is wanted.".
+
+    ```mermaid
+        graph TD
+        MME --> SctpD
+
+        MobilityD -.-> OVS
+        MobilityD -.-> SubscriberDB
+
+        PipelineD -.-> OVS
+        EnvoryController --> Envoy
+    ```
+
+    (Wants = Doted line, Requires = Normal line)
+
+- The option [PartOf](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#PartOf=) includes a set of other services restarted or stopped if the service is restarted or stopped. The following graph shows the `PartOf` configuration for Magma services. For example, `ConnectionD → MME` can be read as "If ConnectionD restarts, then an MME restart will also initiate."
```

Comment:
- `a MME restart`
- Optional: It might again become clearer, if the first sentence is specifically calling the services with some names. (See above)